### DISCUSSION TITLE
Add a typescript example for wrapped daypicker component

### DISCUSSION THREAD

**Discussion by LeunensMichiel** [gpbl/react-day-picker#1411] [General]
Hi! Loving the package.

I was building a custom wrapper for the `Daypicker` component to make it work with `react-hook-form` and implement some custom styling, but I kept getting typescript errors. Was wondering if you could provide an example on the website on how to wrap the daypicker component for custom styling, without altering the logic. Kinda like https://react-select.com/typescript.

Thank you for your work!

**Comment by tjfred35** [comment]
Thanks for the suggestion. Are you using v8 or v7?

**Comment by gpbl** [comment]
I think @LeunensMichiel is talking about v7 – indeed an integration example with `react-hook-form` maybe interesting!

**Comment by LeunensMichiel** [comment]
Well, I used V7 in the past, but couldn't wait to try V8. I've got it to work with `react-hook-form` at beta 29, but after upgrading to beta 35 it broke. Not that it's a problem, it is a beta after all. Hence my suggestion!

**Comment by tjfred35** [comment]
Can you share the code and point out where it broke, possibly via a codesandbox link?
